src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,16 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
```

Comment:
Please remove the redundant empty line after the class declaration (line 3). According to the checklist, we don't need empty lines after class declaration or method signature.